Glade2d/Graphics/Renderer.cs

```diff
@@ -120,7 +138,10 @@ public override void Show()
             if (Scale > 1)
             {
                 GameService.Instance.GameInstance.Profiler.StartTiming("Renderer.Scale");
-                ScaleIntoDisplayBuffer();
+                var sourceBuffer = (BufferRgb565)pixelBuffer;
+                var targetBuffer = (BufferRgb565)display.PixelBuffer;
+                _bufferTransferrer.Transfer(sourceBuffer, targetBuffer, Scale);
```

Comment:
This should probably be moved outside of the scale `if` check,  Otherwise non-scaled games won't be rotated.